experiment/src/cortex/experiment/util.clj

```diff
@@ -24,27 +24,28 @@
 
 (defn- image->observation-data
   "Create an observation from input."
-  [image datatype image-aug-fn]
+  [image datatype colorspace image-aug-fn]
   (patch/image->patch (if image-aug-fn
                         (image-aug-fn image)
                         image)
                       :datatype datatype
-                      :colorspace :gray))
+                      :colorspace colorspace))
 
 
 (defn- file->observation
   "Given a file, returns an observation map (an element of a dataset)."
-  [image-aug-fn datatype num-classes ^File file]
+  [image-aug-fn datatype colorspace num-classes ^File file]
```

Comment:
Instead of this, it may be possible to infer the color space from the results of `i/load-image` below. That way the user of this library doesn't have to know anything about `colorspace`.